plasma/plasma-mvp/specifications/morevp.md

```diff
@@ -0,0 +1,277 @@
+
+In this section, we specify the MoreVP exit protocol and give an intuitive argument toward its correctness. A formal treatment of the protocol is presented in the appendix. 
+
+
+### Definitions
+
+#### Deposit Transaction
+
+A deposit transaction is a special type of transaction that creates a new balance on the Plasma chain. This is similar to a Bitcoin "coinbase" transaction. In practice, this takes the form of a transaction with a single “special” input.
+
+
+#### Spend Transaction
+
+A spend transaction is any transaction that spends a UTXO present on the Plasma chain. Unlike a deposit transaction, a spend transaction does not create any new balance.
+
+
+#### In-flight Transaction
+
+A transaction is considered to be “in-flight” if it has been broadcast but has not yet been included in the Plasma chain. 
+
+
+#### Canonical Transaction
+
+A transaction is “canonical” if none of its inputs were previously spent in any other transaction. The definition of “previously spent” depends on whether or not the transaction in question is included in the Plasma chain.
+
+If the transaction was included in the chain, an input to that transaction would be considered previously spent if another transaction also spending the input was included in the chain *before* the transaction in question. Transaction position is determined by the tuple (block number, transaction index). If the transaction was not included in the chain, an input to that transaction would be considered previously spent if another transaction also spending the input is *known to exist*.
+
+Note that in this second case it’s unimportant whether or not the other transaction is included in the chain. If the other transaction is included in the chain, then the other transaction is clearly included before the transaction in question. If the other transaction is not included in the chain, then we can’t tell which transaction “came first” and therefore simply say that neither is canonical. 
+
+
+#### Exitable Transaction
+
+If a transaction is “exitable,” then a user may attempt to start an exit that references the transaction. All deposit transactions are “exitable” by default. A spend transaction can be called “exitable” if the transaction is correctly formed (e.g. more input value than output value, inputs older than outputs, proper structure) and is properly signed by the owners of the transaction’s inputs. 
+
+
+#### Valid Transaction
+
+A deposit transaction is “valid” if and only if it corresponds exactly to a single deposit on the root chain and is the first deposit transaction to correspond to that deposit. A spend transaction is “valid” if and only if it is exitable, canonical, and only stems from valid transactions (i.e. all transactions in the history are also valid transactions). Note that a transaction would therefore be considered invalid if even a single invalid transaction is present in its history. An exitable transaction is not necessarily a valid transaction.
```

Comment:
when is an exitable transaction not necessarily valid? - per your definition above an exitable transaction is if transaction is correctly formed (inputs older than ouputs, signatures) and presumably those inputs/outputs are valid. if there is a circumstance I would briefly spell out the specific case it's not valid.